queries.sql

```diff
@@ -1,10 +1,59 @@
-/*Queries that provide answers to the questions from all projects.*/
-
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE date_of_birth BETWEEN '1-1-2016' AND '31-12-2019';
-SELECT * FROM animals WHERE neutered = '1' AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = '1';
-SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+BEGIN;
+-- Start a Transaction:
+BEGIN
+
+-- Update "species" to "unspecified" for Animals with Names Ending in "mon":
+UPDATE
+  animals
+SET
+  species = 'unspecified'

```

Comment:
- [ ] In the first transaction, you're asked to update the animal's table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
-- It turns out you only did the first part, but failed to roll back and confirm as per requirements. Kindly carry this out 👍 

### Requirements:
![image](https://github.com/Mike47ip/vet-clinic-database/assets/50990925/6357edad-2798-4d58-8155-211c1dfa41e7)

### Sample code:
```
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

Reply:
- [ ] Please write this query as suggested by the previous reviewer after updating the `animal` table you should verify
```
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```